crates/terminal/src/terminal.rs

```diff
@@ -327,9 +327,15 @@ impl TerminalBuilder {
         completion_tx: Sender<()>,
         cx: &AppContext,
     ) -> Result<TerminalBuilder> {
-        // TODO: Properly set the current locale,
-        env.entry("LC_ALL".to_string())
-            .or_insert_with(|| "en_US.UTF-8".to_string());
+        // If the parent environment doesn't have a locale set
+        // (As is the case when launched from a .app on MacOS),
+        // and the Project doesn't have a locale set (as is the
+        // case when a remote project is opened over SSH), then
```

Comment:
This is not 100% correct, so I'd remove what's in the `()` (see: https://github.com/zed-industries/zed/blob/main/docs/src/environment.md)